client/question/services/code_evaluators/PythonCodeRunnerService.js

```diff
@@ -89,9 +107,13 @@ tie.factory('PythonCodeRunnerService', [
         }
 
         // The run was successful.
-        return CodeEvalResultObjectFactory.create(
-          code, outputLines.join('\n'), observedOutputs,
+        var standardizedOutput = _standardizeOutputInClient(
```

Comment:
Output standardization should be independent of whether the client/server version is being used.